lib/Dancer2/Manual.pod

```diff
@@ -172,51 +244,38 @@ The following will match GET or POST requests to C</myaction>:
         # code
     };
 
-For convenience, any route which matches GET requests will also match HEAD
-requests.
-
-=head2 Route Handlers
-
-The route action is the code reference declared. It can access parameters
-through the specific C<route_parameters>, C<query_parameters>, and 
-C<body_parameters> keywords, which return a L<Hash::MultiValue> object. 
-This hashref is a merge of the route pattern matches and the request params.
+As mentioned previously, any route which matches GET requests will also
+match HEAD requests. This is just convenience for web app developers.
 
-You can have more details about how params are built and how to access them
-in the L<Dancer2::Core::Request> documentation.
-
-=head3 Declaring Routes
+=head2 Route Patterns
 
-To control what happens when a web request is received by your webapp,
-you'll need to declare C<routes>. A route declaration indicates which HTTP
-method(s) it is valid for, the path it matches (e.g. C</foo/bar>), and a
-coderef to execute, which returns the response.
+After the declaration of the HTTP method, your route handler must supply
+a pattern that will be used to match against the path of the incoming request.
 
-    get '/hello/:name' => sub {
-        return "Hi there " . route_parameters->get('name');
-    };
+The simplest route patterns provide an exact match to the requested path:
 
-The above route specifies that, for GET requests to C</hello/...>, the code
-block provided should be executed.
+    get '/foo/bar' => sub {
+      return 'You are on page /foo/bar';
+    }
 
-=head3 Retrieving request parameters
+This route will be activated when a client requests the C</foo/bar>) path.
 
-The L<query_parameters|Dancer2::Manual/query_parameters>,
-L<route_parameters|Dancer2::Manual/route_parameters>, and
-L<body_parameters|Dancer2::Manual/body_parameters> keywords provide
-a L<Hash::MultiValue> result from the three different parameters.
```

Comment:
I think it's important to leave the reference to `Hash::MultiValue`. I'm ok with clarifying the wording, but that's important information being excluded from the manual.

Reply:
Note that I moved reference to `Hash::MultiValue` to the Route Action section below. I forget the thought process behind my moving it. But it's still there.